tgui/packages/tgui/interfaces/HydrogenExchange.js

```diff
@@ -0,0 +1,74 @@
+import { useBackend } from '../backend';
+import { Button, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+export const HydrogenExchange = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { credits, merits, next_merit_rate, credits_to_merits, credit_tax } =
+    data;
+  return (
+    <Window
+      title="Hydrogen Exchange"
+      theme="ntos_terminal"
+      width={400}
+      height={300}
+    >
+      <Window.Content>
+        <Section title={'Stored Currencies'}>
+          <LabeledList>
+            <LabeledList.Item
+              label={
+                'Current credits: ' + credits + ', the current tax rate is'
+              }
+            >
```

Comment:
This will still look a bit weird since all of the text will be greyed out. Maybe look into using `Stack` elements instead? If you need help feel free to ping me on the discord, TGUI can be tough